commonknowledge/leftbookclub
# Social meta data

## Description
Adding social meta data to pages using https://github.com/coderedcorp/wagtail-seo

## Motivation and Context
#39 adding SEO meta data to pages including specific article seo mixin for article type pages. 

## How Can It Be Tested?
Download and run branch locally. Add content for search and social previews in promote panel. View head tag in the page source to see added meta fields.

## Types of changes
